src/Ev.ServiceBus/Management/Wrappers/ReceiverWrapper.cs

```diff
@@ -30,37 +31,33 @@ public class ReceiverWrapper
         _composedOptions = options;
         _parentOptions = parentOptions;
         _provider = provider;
-        _logger = _provider.GetRequiredService<ILogger<ReceiverWrapper>>();
+        _serviceBusClientManagementLogger = _provider.GetRequiredService<ILogger<LoggingExtensions.ServiceBusClientManagement>>();
+        _messageProcessingLogger = _provider.GetRequiredService<ILogger<LoggingExtensions.MessageProcessing>>();
     }
 
     internal string ResourceId => _composedOptions.ResourceId;
     private ServiceBusProcessor? ProcessorClient { get; set; }
 
     public async Task Initialize()
     {
-        _logger.LogInformation("[Ev.ServiceBus] Initialization of receiver client '{ResourceId}': Start", _composedOptions.ResourceId);
```

Comment:
Please keep this log. 
it is only produced during initialization of the app, so the cost of having them is low.
And they are useful to know exactly which resource are initialized